client/task-list/tasks/payments/index.js

```diff
@@ -254,6 +250,21 @@ class Payments extends Component {
 				</div>
 			);
 		}
+
+		if ( method.key === 'wcpay' ) {
```

Comment:
Heads up that I updated the keys in https://github.com/woocommerce/woocommerce-admin/pull/6786 to match the slugs used for payment settings pages.  This one will be `woocommerce_payments`.

The below section is probably also part of the `Action` component.